crates/core/tedge_mapper/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 description = "tedge_mapper is the mapper that translates thin-edge.io data model to c8y/az data model."
 
 [package.metadata.deb]
-depends = "tedge"
+pre-depends = "tedge"
```

Comment:
Don't you have to fix `tedge-agent`, tedge plugins and other tedge debian packages having dependencies on "anything tedge" as well, to use `pre-depends` instead of `depends`?

Reply:
Yes, needs to. I will update the other PR #1035